Dockerfile

```diff
@@ -22,10 +22,11 @@ LABEL "org.opencontainers.image.version"="$IMAGE_VERSION"
 
 # Install the arch specific build of SSM agent *and confirm that it installed* -
 # yum will allow architecture-mismatched packages to not install and consider
-# the run successful.
+# the run successful. 
+# SSM Agent is downloaded from eu-north-1 as this region gets new releases of SSM Agent first.
 COPY ./hashes/ssm ./hashes
 RUN \
-  curl -L "https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/${SSM_AGENT_VERSION}/linux_${ARCH}/amazon-ssm-agent.rpm" \
+  curl -L "https://s3.eu-north-1.amazonaws.com/amazon-ssm-eu-north-1/latest/linux_${ARCH}/amazon-ssm-agent.rpm" \
```

Comment:
I think we need to find what the version structure is in this new bucket so we can continue to reference SSM_AGENT_VERSION, otherwise when a new "latest" is pushed, our hashes won't match, and our build will break (until we're ready to update this package).

Reply:
Thank you for that call out. I've identified a versioned structure, tested it and updated this PR's commit with the change.